docs: update commands documentation (issue 1429) and minor corrections (#1560)
Redocly/redocly-cli: docs/commands/build-docs.md

| --theme.openapi     | string  | Customize your output with [Redoc functionality options](https://redocly.com/docs/api-reference-docs/configuration/functionality/) or [Redoc theming options](https://redocly.com/docs/api-reference-docs/configuration/theming/). |
| --title             | string  | Set the page title.                                                                                                                                                                                                                |
| --version           | boolean | Show version number.                                                                                                                                                                                                               |

## Examples

### Specify API

The `build-docs` command behaves differently depending on how you pass the API to it, and whether the [configuration file](#use-custom-configuration-file) exists.

#### Pass API directly

```bash
redocly build-docs openapi.yaml
```

In this case, the `build-docs` command builds the API description that was passed to the command.

> The configuration file is ignored.

Reviewer: ```suggestion
Even if it exists, the configuration file is ignored.
```

Author: Updated per suggestion.

#### Pass API alias

Instead of a full path, you can use an API name from the `apis` object of your Redocly configuration file.
For example, with a `redocly.yaml` configuration file containing the following entry for `games@v1`:

```yaml Configuration file
apis:
  games@v1:
    root: ./openapi/api-description.json
```

> You can generate a build by giving the API alias name, as shown below:

Reviewer: ```suggestion
You can generate a build by including the API name with the command, as in the following example:
```

Author: Suggestion acknowledged, updated to "You can generate a build by including the API name with the command, as shown in the following example:"

```bash Command
redocly build-docs games@v1
```

In this case, after resolving the path behind the `games@v1` name, `build-docs` generates a build of the `api-description.json` file. For this approach, the Redocly configuration file is mandatory.

> Any additional configurations provided in the file are used by the command.

Reviewer: ```suggestion
Any additional configurations provided in the file are also used by the command.
```

Author: Updated per suggestion.

> ### Use custom configuration file

Reviewer: ```suggestion
### Use alternative file path
```

Author: Updated per suggestion.

Author: Corrected to "filepath" to be consistent with the other command docs and [English convention](https://english.stackexchange.com/questions/428215/filepath-or-file-path).

By default, the CLI tool looks for the [Redocly configuration file](../configuration/index.md) in the current working directory. Use the optional `--config` argument to provide an alternative path to a configuration file.

```bash
redocly build-docs --config=./another/directory/config.yaml
```

> ### Use theme.openapi option

Reviewer: ```suggestion
### Use `theme.openapi` option
```

Author: Updated per suggestion.

> The following command uses the optional `--theme.openapi` argument to build docs with hidden search box:

Reviewer: ```suggestion
The following command uses the optional `--theme.openapi` argument to build docs with the search box hidden:
```

Author: Updated per suggestion.

```bash
redocly build-docs openapi.yaml --theme.openapi.disableSearch
```

> ### Use templateOptions option

Reviewer: ```suggestion
### Use a custom template
```

Author: Updated per suggestion.

> The following command uses the optional `--templateOptions` argument to build docs using a custom Handlebars template and add a custom `templateOptions` object:

Reviewer: ```suggestion
The following command builds docs using a custom Handlebars template and adds metadata to the meta tag in the head of the page using `templateOptions`:
```

Author: Updated per suggestion.
